recipe/meta.yaml

```diff
@@ -1,16 +1,36 @@
 {% set name = "autogluon" %}
-{% set version = "1.0.0" %}
+{% set version = "1.1.1" %}
+
+{% set boto3 = ">=1.10,<2" %}
```

Comment:
After removing duplicates, the packages below are only used once in the recipe, so we can declare their pinnings directly.

- boto3
- numpy
- pandas
- scipy
- psutil
- networkx
- tqdm
- Pillow

Reply:
Nice!